### ISSUE TITLE
Running nx affected --target=test does not seems to work?

### ISSUE THREAD

**Issue by guzmanoj** [NiklasPor/nx-remotecache-azure#13]
I setup a new Nx monorepo using the latest varsion 15.4 + nx-remotecache-azure@latest unfortunately running  
` nx affected --target=test` seems to do nothing. It does work when the app name is specified, so this works: ` nx run my-app:test`

According to the [Nx documentation]( https://nx.dev/packages/nx/documents/affected), using the first command should run the tests for all affected projects, but it looks like the runner is not able to detect them. It's also worth to mention that using the default runner (nx/tasks-runners/default) makes the ` nx affected --target=test` command to work as expected.

![image](https://user-images.githubusercontent.com/45172701/224393468-254e0888-f4ab-4510-b806-e77eec1e7966.png)

**Comment by NiklasPor** [comment]
Interesting, sounds like the API was updated. I'll look into this asap as we're also using this package in production 👀

**Comment by NiklasPor** [comment]
Alright, could you create a repo which reproduces this issues?

For me affected seems to be working without problems (`Nx 15.8-5):

<img width="809" alt="image" src="https://user-images.githubusercontent.com/13211347/224405003-0be1e2e7-81d9-4da6-9c2f-d52dbb641eb5.png">

Although it seems that Nx is stripping the console logs that the cache plugin is sending.

**Comment by guzmanoj** [comment]
Well, this is embarrassing :D
I have tried 2 more times (both MacOS and Windows): it worked perfectly!
Honestly, I don't know what happened before, I remember some issues with the git initialization (Nx exited with error and I had to run git init myself) other than that it's absolutely the same. I have also tried to remove the .git folder and manually initilize the repo... it also worked. I apologize for the confussion!
Have a nice weekend and thank you for such a cool tool!
